subscribe.markdown

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Subscribe
+permalink: /Subscribe
+---
+
+[Request to join our Code Curious Slack](https://rails-girls-berlin.herokuapp.com/)
```

Comment:
This is good, however I hope we can rename the Slack before this goes live. So will need to pause this. Also inside `_config.yml` we should add this to the `header_links` array, so that it shows up in the head.

Reply:
Do you mean to do like this in `_config.yml file`?

```
header_pages:
  - about.markdown
  - events.markdown
  - subsribe.markdown
```